CSLabs.Api/Jobs/VmStatusJob.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Threading.Tasks;
+using CSLabs.Api.Models;
+using CSLabs.Api.Services;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace CSLabs.Api.Jobs
+{
+    public class VmStatusJob : AsyncJob
+    {
+        private readonly IServiceProvider _provider;
+        public VmStatusJob(IServiceProvider provider)
+        {
+            _provider = provider;
+        }
+
+        protected override async Task ExecuteAsync()
+        {
+            using var scope = _provider.CreateScope();
+            using var context = scope.ServiceProvider.GetService<DefaultContext>();
+            
+            // Do job
+            var connectionService = new TestVmConnectionService(context);
+            await connectionService.TestLabVmConnection();
+
```

Comment:
Add the following to the `ServiceProvider.ProvideAppServices` method:

```
 services.AddScoped<TestVmConnectionService>();
```

Then rewrite this section to handle the dependency injection:

```suggestion
            var connectionService = scope.ServiceProvider.GetService<TestVmConnectionService>();
            await connectionService.TestLabVmConnection();
```

Then anything in the constructor of the TestVmConnectionService will be automatically injected.